MHVideoPhotoGallery/ExampleViewControllerCollectionViewInTableView.m

```diff
@@ -206,12 +206,10 @@ -(void)collectionView:(UICollectionView *)collectionView didSelectItemAtIndexPat
                     
                     [self setNeedsStatusBarAppearanceUpdate];
                     
-                    MPMoviePlayerController *player = interactiveTransition.moviePlayer;
+                    AVPlayerViewController *controller = interactiveTransition.moviePlayer;
                     
-                    player.controlStyle = MPMovieControlStyleEmbedded;
-                    player.view.frame = cell.bounds;
-                    player.scalingMode = MPMovieScalingModeAspectFill;
-                    [cell.contentView addSubview:player.view];
+                    controller.view.frame = cell.bounds;
```

Comment:
Do we need to deal with any parent/child view controller containment in this case?

Reply:
This change is almost the same code as before.  I basically renamed player with controller, perhaps I should have left it unchanged. 
I believe the scaling is done by default.  There is already one set of controls to pause and play video in this example.  That's why I decided to leave it off.

Reply:
I have made another change that should fix this issue.